### DISCUSSION TITLE
Other formats (e.g., AAX) beside VST?

### DISCUSSION THREAD

**Discussion by pms967** [ZL-Audio/ZLEqualizer#27] [General]
Hi,

first of all, congratulations and thank you for your great work. Much appreciated!

I've noticed that in the pre-built release you are providing the plugin only in VST format. For me that's just fine, but for Pro-Tools users that's a no-go (PT does not support anything but AAX).

OTOH, Ardour (and Harrison's MixBus) users may prefer LV2 over VST (I don't know if that changed lately, but in the past VST and particularly VST3 support in Ardour was sub-optimal, LV2s worked much better).

AFAIK Juce can produce many more formats, including AAX, AU and LV2 (and, via a [3rd party extension](https://github.com/free-audio/clap-juce-extensions), possibly also the new CLAP).  

Are there any reason why you choose not to provide other formats? Can they be built from the current sources, or there are problems preventing to do so?

If there are no problems, could you be so kind to provide those other formats too in the binary releases? :) 

Thank you again, and keep up the good work!

**Comment by zsliu98** [comment]
Thanks for your suggestions. If JUCE works properly, all those formats can be built from the current sources. I am aware that VST3 and AU may not suit everyone. And it seems that the reputation of Steinberg is on the line. However, I use `pluginval` to carry out automation tests, which only supports VST, VST3, and AU. If a plugin does not pass automation tests, I won't provide it in the releases. Besides that,

- For VST(VST2), it is no longer possible to get approval from Steinberg.
- For AAX, I have to get approval from Avid :) I haven't applied it yet, maybe someday I will...
- For LV2, I have already filled in all necessary fields. If you want to build an LV2 version, add LV2 to line 36 of `CMakeLists.txt` and build the source (target LV2). I would be happy to provide LV2 once `pluginval` supports that!
